addOns/sequence/src/main/resources/org/zaproxy/zap/extension/sequence/resources/Messages.properties

```diff
@@ -20,6 +20,7 @@ sequence.automation.import.sequencecreated = Job {0}: Created sequence with {1}
 sequence.automation.import.summary = Name: {0}, Path: {1}
 sequence.automation.name = Sequence Automation
 sequence.automation.step = Step {0}
+sequence.automation.step.pass = Pass
```

Comment:
We don't need a fail label? That seems strange.

Reply:
Because the failure message comes from the assertion, and thats already i18ned